### ISSUE TITLE
 Request for Per-Game Proton Activation in Steam

### ISSUE THREAD

**Issue by carlosvalin94** [ValveSoftware/steam-for-linux#11245]
**Your system information**
**Steam client version** 8 September 2024
**Distribution (e.g. Ubuntu):** Fedora 40
**Opted into Steam client beta?:** No
**Have you checked for system updates?:** Yes
**GPU:** AMD

**Please describe your issue in as much detail as possible:**

Currently, there is no option to enable Proton on a per-game basis without enabling it globally for all titles. I think it would be beneficial to have a prompt for users when installing a non-compatible game, asking if they would like to enable Proton just for that specific game. 

This would allow users to test individual games that may not be compatible without needing to enable Proton globally, making the experience more flexible and user-friendly.

**Steps for reproducing this issue:**

Launch a non-compatible game on Steam.
Observe the inability to enable Proton for just that title.
Consider the potential improvement of adding a prompt for per-game Proton activation.

**Comment by mwpow3ll** [comment]
TL;DR: You can, but you have to enable the two Global Proton options first. This allows the user to override Steam's prioritization if they need to.

----

Why would you want per-game basis when practically every game is going to need to be run under Proton?

In my opinion, what you're asking for is more work and **not** something I'd like; however, this semi already exists, but I think people just don't understand how Steam prioritizes; let me explain...

1. By default Steam prioritizes the game binaries that run natively on your OS, but once you enable the global Proton settings, the priority is as follows:

- Native OS
- Proton (tested)
- Proton (untested)

So, lets say you want to play:

- Linux native game: Steam will download the Linux binaries

- Windows native game (proton tested): Steam will download the  Proton approved version and the Windows binaries

- Windows native game (proton untested): Steam will download the Proton global version and the Windows binaries

- Linux & Windows native game: Steam will download the Linux binaries, but if you want to run Proton, right-click that game > Properties > Compatibility > Force the use of a specific Steam Play compatibility tool

For example, I started to play Shadowrun again which is natively supported on Linux and Windows, but the native Linux binaries crash on Linux; so, I forced proton for that game.

**Comment by kisak-valve** [comment]
Hello @carlosvalin94, `Currently, there is no option to enable Proton on a per-game basis without enabling it globally for all titles.` this is not true.

In Steam's library view, you can right click on a game or select a game and click the gear icon on the right -> Properties -> Compatibility -> check `Force the use of a specific Steam Play compatibility tool` and select what you want the game to run with. This per-game setting takes priority over Steam's global settings and does not depend on `Enable Steam Play for all other titles` in the global settings.
